iOS-Development/Communities/Readme.md

```diff
@@ -0,0 +1,13 @@
+## Communities you can join 👇
+
+- [Community Classroom](https://discord.io/commclassroom)
```

Comment:
It's `WeMakeDevs` not `Community Classroom`

Reply:
Sure, It was missed from my end.